docs-web/src/main/webapp/src/app/docs/controller/document/DocumentView.js

```diff
@@ -145,11 +145,29 @@ angular.module('docs').controller('DocumentView', function ($scope, $rootScope,
   /**
    * Validate the workflow.
    */
+  $scope.ratingCategories = ['GPA', 'Skills', 'Extracurriculars', 'Experience'].map(function(name) {
```

Comment:
We can as well add GRE to match the criteria we discussed (see the description of #5).

```suggestion
  $scope.ratingCategories = ['GRE', 'GPA', 'Skills', 'Experience', 'Extracurriculars'].map(function(name) {
```